src/DerConverter/Asn/DerAsnType.cs

```diff
@@ -10,6 +10,8 @@ public abstract class DerAsnType
 
         public abstract object Value { get; }
 
+        public bool UseIndefiniteLengthEncoding { get; set; } = false;
```

Comment:
Would be better if we introduce a `LengthEncodingType` enum. Does the encoding part also have to support both modes?

Don't forget adding unit-tests. But maybe I can pick-up this issue as it might require some larger changes if we want to have encoding support.